web/webpack.config.ts

```diff
@@ -70,6 +70,12 @@ const config: Configuration = {
     devMiddleware: {
       writeToDisk: true,
     },
+    client: {
+      overlay: {
+        warnings: false,
```

Comment:
I'm hesitant to turn off the `warnings` and `errors`. I agree, it is annoying to see `WARNING in shared modules react` each time we render. But I'm concerned that other developers will be confused when they don't see the overlays rendered when there is a warning/error and it only appears in the console log. We could include this in the /hack files or as a note in the README.  WDYT?

Reply:
I got too annoyed seeing this warning on every code change (which auto-reloads the page) :grinning: . And the errors about unused imports etc., were also not helpful during development.

I'll revert it and just keep it locally.